unified-runtime/source/adapters/level_zero/v2/command_buffer.hpp

```diff
@@ -32,11 +32,14 @@ struct ur_exp_command_buffer_handle_t_ : public ur_object {
   ur_result_t
   registerExecutionEventUnlocked(ur_event_handle_t nextExecutionEvent);
 
+  const bool isUpdatable = false;
+  const bool isInOrder = true;
+
```

Comment:
Can we keep these next to `isProfilingEnabled` so that all the bool member variables are in the same place for readability, the comment for `isUpdatable` was also not moved with the variable.